### PR TITLE
Compatability with Rails 5.1: build_relation accepts 3 parameters - wit…

### PR DESCRIPTION
**Pull request by iFurt**
…hout `table`

### CODE REVIEW

File: lib/paranoia.rb [hunk 1]
```diff
@@ -263,8 +263,9 @@ def paranoia_sentinel_value
 module ActiveRecord
   module Validations
     module UniquenessParanoiaValidator
-      def build_relation(klass, table, attribute, value)
-        relation = super(klass, table, attribute, value)
+      def build_relation(*args)
```
Comment (BenMorganIO) [inline lib/paranoia.rb]: Just bike shedding 🚲 🏠, but you could do:

```ruby
def build_relation(klass, *args)
  relation = super(klass, *args)
  ...
end
```

This way you don't have to say `klass = args.first`.

**Comment by BenMorganIO** [comment]
Classic JRuby failing...

**Comment by BenMorganIO** [comment]
For 5.0.0.1, it still takes 4 arguments: https://github.com/rails/rails/blob/v5.0.0.1/activerecord/lib/active_record/validations/uniqueness.rb#L53

**But...** Rails 5.1 will not: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/validations/uniqueness.rb#L52

Maybe update your title for Rails 5.1 compatibility?

**Comment by iFurt** [comment]
Sure, thanks for pointing, Ben 👍

**Comment by BenMorganIO** [comment]
Also, thank you for this!

**Comment by BenMorganIO** [comment]
Thanks. Could you rebase and squash them both into one commit and update the message to read 5.1 as well?

**Comment by iFurt** [comment]
Seems that I've messed up with rebase..

**Pull request closed without merging**